pandas/info.py

```diff
@@ -1,20 +0,0 @@
-"""
-pandas - a powerful data analysis and manipulation library for Python
-=====================================================================
-
-See http://pandas.pydata.org/ for full documentation. Otherwise, see the
```

Comment:
I just did this
```
In [1]: pd?
Type:        module
String form: <module 'pandas' from '/Users/jreback/pandas/pandas/__init__.py'>
File:        ~/pandas/pandas/__init__.py
Docstring:  
pandas - a powerful data analysis and manipulation library for Python
=====================================================================

**pandas** is a Python package providing fast, flexible, and expressive data
structures designed to make working with "relational" or "labeled" data both
easy and intuitive. It aims to be the fundamental high-level building block for
doing practical, **real world** data analysis in Python. Additionally, it has
the broader goal of becoming **the most powerful and flexible open source data
analysis / manipulation tool available in any language**. It is already well on
its way toward this goal.

Main Features
-------------
Here are just a few of the things that pandas does well:

  - Easy handling of missing data in floating point as well as
    non-floating point data
  - Size mutability: columns can be inserted and deleted from
    DataFrame and higher dimensional objects
  - Automatic and explicit data alignment: objects can  be
    explicitly aligned to a set of labels, or the user can simply
    ignore the labels and let `Series`, `DataFrame`, etc. automatically
    align the data for you in computations
  - Powerful, flexible group by functionality to perform
    split-apply-combine operations on data sets, for both aggregating
    and transforming data
  - Make it easy to convert ragged, differently-indexed data in
    other Python and NumPy data structures into DataFrame objects
  - Intelligent label-based slicing, fancy indexing, and subsetting of
    large data sets
  - Intuitive merging and joining data sets
  - Flexible reshaping and pivoting of data sets
  - Hierarchical labeling of axes (possible to have multiple
    labels per tick)
  - Robust IO tools for loading data from flat files
    (CSV and delimited), Excel files, databases,
    and saving/loading data from the ultrafast HDF5 format
  - Time series-specific functionality: date range
    generation and frequency conversion, moving window statistics,
    moving window linear regressions, date shifting and lagging, etc.
```

as the README.md has all kinds of html and such (and is too long)